File: laba0/pltr.py
import math
a= int(input('Введите число a'))
b= int(input('Введите число b'))
c= int(input('Введите число c'))
P= int((a+b+c)/2)
S = float(math.sqrt(P*(P-a)*(P-b)*(P-c)))
print (S)

File: laba0/Pltr.pas
program pltr;
var a,b,c,P,S:real;
begin
Writeln ('Введите 3 числа');
Readln (a,b,c);
P:= (a+b+c)/2;
S:= sqrt(P*(P-a)*(P-b)*(P-c));
Writeln (S);
end.